core/object/message_queue.h

```diff
@@ -33,26 +33,44 @@
 
 #include "core/object/object_id.h"
 #include "core/os/thread_safe.h"
+#include "core/templates/local_vector.h"
+#include "core/templates/paged_allocator.h"
 #include "core/variant/variant.h"
-
 class Object;
 
-class MessageQueue {
-	_THREAD_SAFE_CLASS_
-
+class CallQueue {
+public:
 	enum {
-		DEFAULT_QUEUE_SIZE_KB = 4096
+		QUEUE_PAGE_SIZE = 4096 // PAGE_SIZE seems like a reserved identifier on some Linux.
```

Comment:
I don't know if we'll really care about this trivia once it's merged and we wouldn't consider renaming it anyway :P